benchmarks/asv/app_suite.py

```diff
@@ -31,3 +37,202 @@ def time_hydrogen(self, n_steps, optimize):
         hyperparams = {'n_steps': n_steps,
                        'optimize': optimize}
         benchmark_vqe(hyperparams)
+
+    def peakmem_hydrogen(self, n_steps, optimize):
+        """Benchmark the peak memory usage of the VQE algorithm with the UCCSD ansatz for computing
+         the ground state energy of the hydrogen molecule."""
+        hyperparams = {'n_steps': n_steps,
+                       'optimize': optimize}
+        benchmark_vqe(hyperparams)
+
+
+class VQE_heavy:
+    """Benchmark the VQE algorithm using different grouping options for the lithium hydride molecule
+     with 2 active electrons and 8 active spin-orbitals. The sto-3g basis set and UCCSD ansatz are
+     used."""
+
+    params = ([3], [False, True])
+    param_names = ['n_steps', 'optimize']
```

Comment:
Would one even need this first parameter?

Reply:
No, we do not need it. I kept it in case we would like to have multiple parameters. We can simply get rid of these _params_ here.

Reply:
Would peakmem change with `n_steps`?

If additional steps don't add much additional memory, maybe we should run peakmem with just one step.

Reply:
I agree that one step should probably be enough